features/facebookImport/src/components/dataStructureMiniStory/dataStructureMiniStory.jsx

```diff
@@ -84,12 +83,15 @@ const DataStructureMiniStory = ({ data }) => {
                     onBubbleClick={handleBubbleClick}
                 />
             </div>
-            <ChartButtons
-                buttonsContent={dataWithTotal.map((d) => {
+            <Chips
+                chipsContent={dataWithTotal.map((d) => {
```

Comment:
I see, so in some cases, the button will only have the title, and show that as ID, is that correct? Going back :arrow_up: